### ISSUE TITLE
Tags not being updated in Radarr

### ISSUE THREAD

**Issue by Tharic99** [Adman1020/Elsewherr#6]
Hi there,

We've talked over on Reddit. I'm running Radarr Version 4.2.1.6468 and the most recent wget from your Elsewherr repository here. 

None of my tags are being updated in Radarr though and I'm getting "Response [200]" results from the API. I've tested a few things using the API generator for Radarr itself over on https://radarr.video/docs/api/ and putting my IP and API info in and I'm able to get results back from the API there, but when things run through Elsewherr, I'm getting the 200 errors.

My full log file and config.yml are attached, I ran things with the -d debug option but it crashed out during running, so the log ends abruptly as well. However, here is the code from running it and where it crashed on the error as well. 

If it helps any, I've got 1762 movies in Radarr as well, not sure if that's a much larger number than you or not though.

```
root@6daf3591959c:/chronos/elsewherr# python elsewherr.py -d
Traceback (most recent call last):
  File "elsewherr.py", line 79, in <module>
    logging.debug(f'Total Providers: {len(tmdbProviders["results"])}')
KeyError: 'results'
root@6daf3591959c:/chronos/elsewherr# 
```

My log and config.yml are linked here. It's a secure link and the password is your reddit username. They'll also expire after 14 days as well, just in case. 

https://safenote.co/r/62f3e040c22f38@00897832

Thanks

**Comment by LiamAird** [comment]
I also ran into this issue with a linux install. 

So what I ended up doing was cloning it down to my windows desktop, installing python3 through windows store. 
Then `python.exe elsewherr.py` works fine for me. 

My config yaml just includes my https link to my radarr installation` radarrUrl: https://url.domain`
This works for me. I did not get it to work on a linux machine locally with the radarr install.

**Comment by Tharic99** [comment]
@Adman1020 

Issue has been resolved. I spent some time this morning diving through the debug logs and code, running the API calls directly on my own in a browser, etc.

The config.yaml.example had the Radarr setting as "radarrUrl: http://localhost:7878"

However, that did not work for me. This resolution was as simple as changing that to "radarrUrl: http://localhost:7878/radarr" and the script is working now.
